src/styles/colors.css

```diff
@@ -117,4 +122,4 @@ Some are best exposed as selectors. Here's the list:
 
 .color-text-on-dark {
   color: var(--color-text-on-dark);
-}
+}
```

Comment:
Was this intentional?

Reply:
i fundamentally don't understand why, but prettier did it this way. i tried multiple times to stop it 😕

Reply:
lol never mind somehow my VS Code had its newline thing setup to automatically remove trailing newlines instead of automatically adding them... fixed in the package bump this got rolled into.